src/components/List/List.jsx

```diff
@@ -59,38 +108,35 @@ export const List = ({ token }) => {
             </button>
           </div>
           <ul className="collection-list">
-            {value.docs
+            {items
               .filter((doc) =>
-                doc
-                  .data()
-                  .item.toLowerCase()
-                  .includes(filterText.toLowerCase()),
+                doc.item.toLowerCase().includes(filterText.toLowerCase()),
               )
               .map((doc) => (
-                <li key={doc.id}>
+                <li key={doc.id} className={`${setColor(doc)}`}>
                   <input
                     type="checkbox"
-                    checked={calcTimeDiff(doc.data().last_purchased_date)}
-                    disabled={calcTimeDiff(doc.data().last_purchased_date)}
+                    checked={calcTimeDiff(doc.last_purchased_date)}
+                    disabled={calcTimeDiff(doc.last_purchased_date)}
                     name={doc.id}
                     id={doc.id}
                     onClick={(e) => handleClick(doc, e)}
                     onChange={(e) => handleClick(doc, e)}
+                    aria-label={setColor(doc)}
```

Comment:
To belabor the point, here's an example showing why the function name probably shouldn't mention "color". The label isn't about the color; it's about the group/category.